x/payment/client/testutil/integration_test.go

```diff
@@ -130,8 +132,14 @@ func (s *IntegrationTestSuite) TestSubmitWirePayForMessage() {
 					"test: %s, output\n:", tc.name, out.String())
 
 				events := txResp.Logs[0].GetEvents()
-				for i := 0; i < len(events); i++ {
-					s.Equal("/payment.MsgPayForMessage", events[i].GetAttributes()[0].Value)
+				for _, e := range events {
+					switch e.Type {
+					case etypes.AttributeKeySpender:
```

Comment:
here I think we're looking for `EventTypePayForMessage` instead of the attribute key